docs/api/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
This is autogenerated by sphinx. I'd prefer not to write anything by myself, and just use the tool.